Numerical/Trailing_Zeroes.py

```diff
@@ -0,0 +1,31 @@
+# Input: number = 3
+# Output: 0
+# Explanation: 3! = 6, no trailing zero.
+
+# Input: number = 5
+# Output: 1
+# Explanation: 5! = 120

```

Comment:
Add the sample output after the code

Reply:
I'm sorry I didn't understand that!
I'm assuming you want me to move the selected comments below the code, is it correct?

Reply:
YES

Reply:
Hey, Actually I deleted the fork yesterday by mistake would you mind if I come with a new PR?

Reply:
I've solved it Via #228 
Can you please review it now!